Sources/protoc-gen-grpc-swift/main.swift

```diff
@@ -65,14 +65,9 @@ func outputFileName(
   component: String,
   fileDescriptor: FileDescriptor,
   fileNamingOption: FileNaming,
-  binaryFile: Bool
+  extensionType: String
 ) -> String {
-    let ext: String
-    if binaryFile {
-        ext = "." + component + ".txt"
-    } else {
-        ext = "." + component + ".swift"
-    }
+  let ext = "." + component + extensionType
```

Comment:
The "." isn't usually included in the extension, so instead of passing it in `extensionType` we should be adding it here.